kwalify/component/v3.1.0.yaml

```diff
@@ -0,0 +1,125 @@
+type: map
+mapping:
+  name:
+    type: str
+    required: true
+  key:
+    type: str
+  system:
+    type: str
+    required: false
+  schema_version:
+    type: str
+    required: true
+  documentation_complete:
+    type: bool
+  responsible_role:
+    type: str
+  references:
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          name:
+            type: str
+            required: true
+          path:
+            type: str
+          type:
+            type: str
+            required: true
+  verifications:
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          key:
+            type: str
+            required: true
+          name:
+            type: str
+            required: true
+          path:
+            type: str
+          type:
+            type: str
+            required: true
+          description:
+            type: str
+          test_passed:
+            type: bool
+          last_run:
+            type: any
+  satisfies:
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          standard_key:
+            type: text
+            required: true
+          control_key:
+            type: text
+            required: true
+          narrative:
+            required: true
+            type: seq
+            sequence:
+              - type: map
+                mapping:
+                  key:
+                    type: str
+                  text:
+                    type: str
+                    required: true
+          control_origin:
+            type: str
+          control_origins:
+            type: seq
+            sequence:
+              - type: map
+                mapping:
+                  text:
+                    type: str
+          parameters:
+            type: seq
+            sequence:
+              - type: map
+                mapping:
+                  key:
+                    type: str
+                    required: true
+                  text:
+                    type: str
+                    required: true
+          implementation_status:
+            type: str
+            enum:
+              - partial
+              - complete
+              - planned
+              - none
```

Comment:
Maybe we add a comment or something saying this is deprecated, or at least that `implementation_statuses` is an alternative?